# List of OpenMS Publications

## 2023
- Guo J, Huan T. Mechanistic Understanding of the Discrepancies between Common Peak Picking Algorithms in Liquid Chromatography-Mass Spectrometry-Based Metabolomics.. Analytical chemistry. 2023. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Wein S. Analysis of RNA Sequences and Modifications Using NASE.. Methods in molecular biology (Clifton, N.J.). 2023. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2022
- Uszkoreit J, Barkovits K, Pacharra S, Pfeiffer K, Steinbach S, Marcus K, Eisenacher M. Dataset containing physiological amounts of spike-in proteins into murine C2C12 background as a ground truth quantitative LC-MS/MS reference.. Data in brief. 2022. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Jeong K, Kim J, Kohlbacher O. Mass Deconvolution of Top-Down Mass Spectrometry Datasets by FLASHDeconv.. Methods in molecular biology (Clifton, N.J.). 2022. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Sénécaut N, Poulain P, Lignières L, Terrier S, Legros V, Chevreux G, Lelandais G, Camadro JM. Quantitative Proteomics in Yeast : From bSLIM and Proteome Discoverer Outputs to Graphical Assessment of the Significance of Protein Quantification Scores.. Methods in molecular biology (Clifton, N.J.). 2022. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2021
- Delabriere A, Warmer P, Brennsteiner V, Zamboni N. SLAW: A Scalable and Self-Optimizing Processing Workflow for Untargeted LC-MS.. Analytical chemistry. 2021. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Svecla M, Garrone G, Faré F, Aletti G, Norata GD, Beretta G. DDASSQ: An open-source, multiple peptide sequencing strategy for label free quantification based on an OpenMS pipeline in the KNIME analytics platform.. Proteomics. 2021. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Bichmann L, Gupta S, Rosenberger G, Kuchenbecker L, Sachsenberg T, Ewels P, Alka O, Pfeuffer J, Kohlbacher O, Röst H. DIAproteomics: A Multifunctional Data Analysis Pipeline for Data-Independent Acquisition Proteomics and Peptidomics.. Journal of proteome research. 2021. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Reutrakul S, Chen H, Chirakalwasan N, Charoensri S, Wanitcharoenkul E, Amnakkittikul S, Saetung S, Layden BT, Chlipala GE. Metabolomic profile associated with obstructive sleep apnoea severity in obese pregnant women with gestational diabetes mellitus: A pilot study.. Journal of sleep research. 2021. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Sénécaut N, Alves G, Weisser H, Lignières L, Terrier S, Yang-Crosson L, Poulain P, Lelandais G, Yu YK, Camadro JM. Novel Insights into Quantitative Proteomics from an Innovative Bottom-Up Simple Light Isotope Metabolic (bSLIM) Labeling Data Processing Strategy.. Journal of proteome research. 2021. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2020
- Netz E, Dijkstra TMH, Sachsenberg T, Zimmermann L, Walzer M, Monecke T, Ficner R, Dybkov O, Urlaub H, Kohlbacher O. OpenPepXL: An Open-Source Tool for Sensitive Identification of Cross-Linked Peptides in XL-MS.. Molecular & cellular proteomics : MCP. 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Huang T, Choi M, Tzouros M, Golling S, Pandya NJ, Banfai B, Dunkley T, Vitek O. MSstatsTMT: Statistical Detection of Differentially Abundant Proteins in Experiments with Isobaric Labeling and Multiple Mixtures.. Molecular & cellular proteomics : MCP. 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Ramachandran S, Thomas T. A Frequency-Based Approach to Predict the Low-Energy Collision-Induced Dissociation Fragmentation Spectra.. ACS omega. 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Jeong K, Kim J, Gaikwad M, Hidayah SN, Heikaus L, Schlüter H, Kohlbacher O. FLASHDeconv: Ultrafast, High-Quality Feature Deconvolution for Top-Down Proteomics.. Cell systems. 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Pfeuffer J, Sachsenberg T, Dijkstra TMH, Serang O, Reinert K, Kohlbacher O. EPIFANY: A Method for Efficient High-Confidence Protein Inference.. Journal of proteome research. 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Rurik M, Alka O, Aicheler F, Kohlbacher O. Metabolomics Data Processing Using OpenMS.. Methods in molecular biology (Clifton, N.J.). 2020. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2019
- Bichmann L, Nelde A, Ghosh M, Heumos L, Mohr C, Peltzer A, Kuchenbecker L, Sachsenberg T, Walz JS, Stevanović S, Rammensee HG, Kohlbacher O. MHCquant: Automated and Reproducible Data Analysis for Immunopeptidomics.. Journal of proteome research. 2019. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Li X, Nakayama K, Goto T, Akamatsu S, Shimizu K, Ogawa O, Inoue T. Data processing on a comparative evaluation of the extraction and analysis procedures for urinary phospholipid and lysophospholipid using MALDI-TOF/MS.. Data in brief. 2019. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2018
- Goldfarb D, Lafferty MJ, Herring LE, Wang W, Major MB. Approximating Isotope Distributions of Biomolecule Fragments.. ACS omega. 2018. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Pioch M, Hoffmann M, Pralow A, Reichl U, Rapp E. glyXtool(MS): An Open-Source Pipeline for Semiautomated Analysis of Glycopeptide Mass Spectrometry Data.. Analytical chemistry. 2018. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2017
- Koch M, Duigou T, Carbonell P, Faulon JL. Molecular structures enumeration and virtual screening in the chemical space with RetroPath2.0.. Journal of cheminformatics. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Flett FJ, Sachsenberg T, Kohlbacher O, Mackay CL, Interthal H. Differential Enzymatic (16)O/(18)O Labeling for the Detection of Cross-Linked Nucleic Acid-Protein Heteroconjugates.. Analytical chemistry. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Schlaffner CN, Pirklbauer GJ, Bender A, Choudhary JS. Fast, Quantitative and Variant Enabled Mapping of Peptides to Genomes.. Cell systems. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Weisser H, Choudhary JS. Targeted Feature Detection for Data-Dependent Shotgun Proteomics.. Journal of proteome research. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Pfeuffer J, Sachsenberg T, Alka O, Walzer M, Fillbrunn A, Nilse L, Schilling O, Reinert K, Kohlbacher O. OpenMS - A platform for reproducible analysis of mass spectrometry data.. Journal of biotechnology. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Audain E, Uszkoreit J, Sachsenberg T, Pfeuffer J, Liang X, Hermjakob H, Sanchez A, Eisenacher M, Reinert K, Tabb DL, Kohlbacher O, Perez-Riverol Y. In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics.. Journal of proteomics. 2017. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2016
- Codrea MC, Nahnsen S. Platforms and Pipelines for Proteomics Data Analysis and Management.. Advances in experimental medicine and biology. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Zühlke M, Riebe D, Beitz T, Löhmannsröben HG, Andreotti S, Reinert K, Zenichowski K, Diener M. High-performance liquid chromatography with electrospray ionization ion mobility spectrometry: Characterization, data management, and applications.. Journal of separation science. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Weisser H, Wright JC, Mudge JM, Gutenbrunner P, Choudhary JS. Flexible Data Analysis Pipeline for High-Confidence Proteogenomics.. Journal of proteome research. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Röst HL, Sachsenberg T, Aiche S, Bielow C, Weisser H, Aicheler F, Andreotti S, Ehrlich HC, Gutenbrunner P, Kenar E, Liang X, Nahnsen S, Nilse L, Pfeuffer J, Rosenberger G, Rurik M, Schmitt U, Veit J, Walzer M, Wojnar D, Wolski WE, Schilling O, Choudhary JS, Malmström L, Aebersold R, Reinert K, Kohlbacher O. OpenMS: a flexible open-source software platform for mass spectrometry data analysis.. Nature methods. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Nilse L, Avci D, Heisterkamp P, Serang O, Lemberg MK, Schilling O. Yeast membrane proteomics using leucine metabolic labelling: Bioinformatic data processing and exemplary application to the ER-intramembrane protease Ypf1.. Biochimica et biophysica acta. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Burns NK, Andrighetto LM, Conlan XA, Purcell SD, Barnett NW, Denning J, Francis PS, Stevenson PG. Blind column selection protocol for two-dimensional high performance liquid chromatography.. Talanta. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Khoonsari PE, Häggmark A, Lönnberg M, Mikus M, Kilander L, Lannfelt L, Bergquist J, Ingelsson M, Nilsson P, Kultima K, Shevchenko G. Analysis of the Cerebrospinal Fluid Proteome in Alzheimer's Disease.. PloS one. 2016. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2015
- Winkler R. An evolving computational platform for biological mass spectrometry: workflows, statistics and data mining with MASSyPup64.. PeerJ. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Ranninger C, Rurik M, Limonciel A, Ruzek S, Reischl R, Wilmes A, Jennings P, Hewitt P, Dekant W, Kohlbacher O, Huber CG. Nephron Toxicity Profiling via Untargeted Metabolome Analysis Employing a High Performance Liquid Chromatography-Mass Spectrometry-based Experimental and Computational Pipeline.. The Journal of biological chemistry. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Qi D, Zhang H, Fan J, Perkins S, Pisconti A, Simpson DM, Bessant C, Hubbard S, Jones AR. The mzqLibrary--An open source Java library supporting the HUPO-PSI quantitative proteomics standard.. Proteomics. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Röst HL, Schmitt U, Aebersold R, Malmström L. Fast and Efficient XML Data Access for Next-Generation Mass Spectrometry.. PloS one. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Wang Y, Yang F, Wu P, Bu D, Sun S. OpenMS-Simulator: an open-source software for theoretical tandem mass spectrum prediction.. BMC bioinformatics. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Aiche S, Sachsenberg T, Kenar E, Walzer M, Wiswedel B, Kristl T, Boyles M, Duschl A, Huber CG, Berthold MR, Reinert K, Kohlbacher O. Workflows for automated downstream data analysis and visualization in large-scale computational mass spectrometry.. Proteomics. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Sachsenberg T, Herbst FA, Taubert M, Kermer R, Jehmlich N, von Bergen M, Seifert J, Kohlbacher O. MetaProSIP: automated inference of stable isotope incorporation rates in proteins for functional metaproteomics.. Journal of proteome research. 2015. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2014
- Aoshima K, Takahashi K, Ikawa M, Kimura T, Fukuda M, Tanaka S, Parry HE, Fujita Y, Yoshizawa AC, Utsunomiya S, Kajihara S, Tanaka K, Oda Y. A simple peak detection and label-free quantitation algorithm for chromatography-mass spectrometry.. BMC bioinformatics. 2014. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Kramer K, Sachsenberg T, Beckmann BM, Qamar S, Boon KL, Hentze MW, Kohlbacher O, Urlaub H. Photo-cross-linking and high-resolution mass spectrometry for assignment of RNA-binding sites in RNA-binding proteins.. Nature methods. 2014. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Röst HL, Schmitt U, Aebersold R, Malmström L. pyOpenMS: a Python-based interface to the OpenMS mass-spectrometry algorithm library.. Proteomics. 2014. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Kenar E, Franken H, Forcisi S, Wörmann K, Häring HU, Lehmann R, Schmitt-Kopplin P, Zell A, Kohlbacher O. Automated label-free quantification of metabolites from liquid chromatography-mass spectrometry data.. Molecular & cellular proteomics : MCP. 2014. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Perez-Riverol Y, Wang R, Hermjakob H, Müller M, Vesada V, Vizcaíno JA. Open source libraries and frameworks for mass spectrometry based proteomics: a developer's perspective.. Biochimica et biophysica acta. 2014. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2013
- Benjamin AM, Thompson JW, Soderblom EJ, Geromanos SJ, Henao R, Kraus VB, Moseley MA, Lucas JE. A flexible statistical model for alignment of label-free proteomics data--incorporating ion mobility and product ion information.. BMC bioinformatics. 2013. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Novák J, Sachsenberg T, Hoksza D, Skopal T, Kohlbacher O. On comparison of SimTandem with state-of-the-art peptide identification tools, efficiency of precursor mass filter and dealing with variable modifications.. Journal of integrative bioinformatics. 2013. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Zerck A, Nordhoff E, Lehrach H, Reinert K. Optimal precursor ion selection for LC-MALDI MS/MS.. BMC bioinformatics. 2013. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Kiefer P, Schmitt U, Vorholt JA. eMZed: an open source framework in Python for rapid and interactive development of LC/MS data analysis workflows.. Bioinformatics (Oxford, England). 2013. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Weisser H, Nahnsen S, Grossmann J, Nilse L, Quandt A, Brauer H, Sturm M, Kenar E, Kohlbacher O, Aebersold R, Malmström L. An automated pipeline for high-throughput label-free quantitative proteomics.. Journal of proteome research. 2013. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2012
- Nahnsen S, Kohlbacher O. In silico design of targeted SRM-based experiments.. BMC bioinformatics. 2012. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Junker J, Bielow C, Bertsch A, Sturm M, Reinert K, Kohlbacher O. TOPPAS: a graphical workflow editor for the analysis of high-throughput proteomics data.. Journal of proteome research. 2012. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Hoekman B, Breitling R, Suits F, Bischoff R, Horvatovich P. msCompare: a framework for quantitative analysis of label-free LC-MS data for comparative candidate biomarker studies.. Molecular & cellular proteomics : MCP. 2012. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Andreotti S, Klau GW, Reinert K. Antilope--a Lagrangian relaxation approach to the de novo peptide sequencing problem.. IEEE/ACM transactions on computational biology and bioinformatics. 2012. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2011
- Ballardini R, Benevento M, Arrigoni G, Pattini L, Roda A. MassUntangler: a novel alignment tool for label-free liquid chromatography-mass spectrometry proteomic data.. Journal of chromatography. A. 2011. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Nahnsen S, Bertsch A, Rahnenführer J, Nordheim A, Kohlbacher O. Probabilistic consensus scoring improves tandem mass spectrometry peptide identification.. Journal of proteome research. 2011. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Bertsch A, Gröpl C, Reinert K, Kohlbacher O. OpenMS and TOPP: open source software for LC-MS data analysis.. Methods in molecular biology (Clifton, N.J.). 2011. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2010
- Sun Y, Zhang J, Braga-Neto U, Dougherty ER. BPDA - a Bayesian peptide detection algorithm for mass spectrometry.. BMC bioinformatics. 2010. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Reinert K, Kohlbacher O. OpenMS and TOPP: open source software for LC-MS data analysis.. Methods in molecular biology (Clifton, N.J.). 2010. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2009
- Hussong R, Gregorius B, Tholey A, Hildebrandt A. Highly accelerated feature detection in proteomics data sets using modern graphics processing units.. Bioinformatics (Oxford, England). 2009. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Sturm M, Kohlbacher O. TOPPView: an open-source viewer for mass spectrometry data.. Journal of proteome research. 2009. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2008
- Sturm M, Bertsch A, Gröpl C, Hildebrandt A, Hussong R, Lange E, Pfeifer N, Schulz-Trieglaff O, Zerck A, Reinert K, Kohlbacher O. OpenMS - an open-source software framework for mass spectrometry.. BMC bioinformatics. 2008. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2007
- Lange E, Gröpl C, Schulz-Trieglaff O, Leinenbach A, Huber C, Reinert K. A geometric approach for the alignment of liquid chromatography-mass spectrometry data.. Bioinformatics (Oxford, England). 2007. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)

- Kohlbacher O, Reinert K, Gröpl C, Lange E, Pfeifer N, Schulz-Trieglaff O, Sturm M. TOPP--the OpenMS proteomics pipeline.. Bioinformatics (Oxford, England). 2007. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
## 2006
- Lange E, Gröpl C, Reinert K, Kohlbacher O, Hildebrandt A. High-accuracy peak picking of proteomics data using wavelet techniques.. Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing. 2006. [Link](https://www.ncbi.nlm.nih.gov/pubmed/%7Bpub_id%7D)
***
